### ISSUE TITLE
Installation Fails with Python 3.12

### ISSUE THREAD

**Issue by nnikolov-intel** [tianocore/edk2-edkrepo#241]
I have Python 3.10 and Python 3.12 on my system and the installation fails.
3.12 breaks the installation - this more in line with "looking in the future" than something urgent.
Installation log
```
Found Python 3.10.11
Found Python 3.12.3
Found Python 2.7.15
Git Version: 2.44.0
Running git lfs install
Running git config --global core.longpaths true
Searching for existing EdkRepo installations...
Determining currently installed Python packages...
Uninstalling current version of edkrepo
Version 4.0.0 of package smmap is already installed
Version 4.0.7 of package gitdb is already installed
Version 3.1.14 of package GitPython is already installed
Version 0.4.4 of package colorama is already installed
Installing edkrepo version 3.2.0
Windows directory is in %PATH%.
Creating edkrepo launcher in Git Bash...
Creating python launcher in Git Bash...
Creating python3 launcher in Git Bash...
Creating python2 launcher in Git Bash...
Installing EdkRepo command completion...
Copied win_edkrepo_prompt.sh
C:\WINDOWS\edkrepo.exe
C:\Program Files\Git\etc\profile.d\edkrepo_completions.sh
Traceback (most recent call last):
File "<frozen runpy>", line 198, in _run_module_as_main
File "<frozen runpy>", line 88, in _run_code
File "C:\WINDOWS\edkrepo.exe\__main__.py", line 4, in <module>
File "C:\Program Files\Python312\Lib\site-packages\edkrepo\edkrepo_entry_point.py", line 16, in <module>
import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
Installation Error:
System.InvalidOperationException: generate-command-completion-script failed with status 1
at TianoCore.EdkRepoInstaller.InstallWorker.PerformInstall(Action`2 ReportComplete, Action`1 ReportProgress, Action`1 AllowCancel, Func`1 CancelPending)
at TianoCore.EdkRepoInstaller.ProgressWindow.WorkerThread()
Installation Failed.
```

**Comment by ndhaller** [comment]
I've been able to use Python 3.12.3 if integrating this pull request into the installer.
https://github.com/tianocore/edk2-edkrepo/pull/240

**Comment by nate-desimone** [comment]
Fixed in 0be4a78
